improver_tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToProbabilities.py

```diff
@@ -175,11 +176,64 @@ def test_threshold_above_cube(self):
         """Test that the expected probabilities are returned for a cube in
         which they are calculated above the thresholds."""
 
-        expected = (np.ones((3, 3, 3)) * [0.75, 0.5, 0.25]).T
         result = Plugin()._location_and_scale_parameters_to_probabilities(
             self.location_parameter_values, self.scale_parameter_values,
             self.template_cube)
-        np.testing.assert_allclose(result.data, expected, rtol=1.e-4)
+        np.testing.assert_allclose(result.data, self.expected, rtol=1.e-4)
+
+    def test_with_location_mask(self):
+        """Test that the plugin gets the correct data out when you put in
+        masked location parameter data."""
+        mask = np.array([[1, 0, 0], [0, 0, 0], [0, 0, 0]])
+        expected_mask = np.array([[[1, 0, 0], [0, 0, 0], [0, 0, 0]],
+                                  [[1, 0, 0], [0, 0, 0], [0, 0, 0]],
+                                  [[1, 0, 0], [0, 0, 0], [0, 0, 0]]])
```

Comment:
Can be re written using a broadcast to reduce the repetition.
```
        expected_mask = np.broadcast_to(mask, (3,3,3))
```
It takes the original mask and transforms it into the shape given.